Build Docker images (base, qns) via CI (#258)
cloudflare/quiche: docker/README.md

# Docker images

> You can find quiche docker images in https://hub.docker.com/repository/docker/cloudflare/quiche

Reviewer: ```suggestion
You can find tha quiche Docker images on [Docker Hub](https://hub.docker.com/repository/docker/cloudflare/quiche).
```

Reviewer 2: typo in suggestion

Reviewer: uh, where? I can't see it.

Reviewer 2: `You can find tha quiche Docker images`

Reviewer: I may be daft, but I still don't see it :eyes: or maybe there's some grammatical rule I'm missing.

Reviewer 2: s/tha/the

Reviewer: Oh lol, fixed, thanks :rofl:

## How to build

> In this directory, simply run

Reviewer: ```suggestion
In order to build the Docker images, simply run the following command in this directory:
```

```
$ make all
```

> To build docker images.

Reviewer: ```suggestion
```
